scraping/heavenly_scraper.py

```diff
@@ -0,0 +1,55 @@
+def ski_scraper(url):
```

Comment:
This entire directory should be removed.

Reply:
Removed directory.